### PR TITLE
Feature 2: My work Section and About Me Section 

### PR DESCRIPTION
**Pull request by Swabri22**
Edited Index.html to add the other parts of the portfolio, also made changes to the css

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -63,5 +63,211 @@ <h1>
...
+        <h3 class="feature-header">Languages</h3>
+        <div id="lang-id" class="lang-card-specs">
+          <ul class="d-flex t-center">
+            <li>JavaScript</li>
+            <li>Ruby</li>
+            <li>HTML</li>
+            <li>CSS</li>
+          </ul>
+        </div>
+      </div>
```
Comment (BenMukebo) [inline index.html:205]: - [x] Kindly, change the main `div` tag to a `ul` tag, in this case, its child should be changed to `li`, but remember, the technologies have to stay in a ul container and li for each technology, so do not change that section (line 198-203), do the same for all cards in the about me section. > Ref:`id: "feature"`

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi **@Swabri22,**

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!

### To highlight 💪🏻

- No linter errors. ✔️ 
- Good commit messages ✔️ 
- Professional README ✔️ 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [x] Please, Write a meaningful title for your pull request and always add a short summary of changes introduced in the pull request. you can check the  [Professional repo - basic rules](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)

Cheers and Happy coding!👏👏👏 💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Whoistolu) [reply index.html:205]: 👍🏽 All good now.

**Review by Whoistolu — APPROVED** [review]
### Hi @Swabri22,

Your project is complete!  There is nothing else to say other than... It's time to merge.

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**